src/render.js

```diff
@@ -30,6 +39,15 @@ export default function render(
     getAllByName: getAllByName(instance),
     getAllByText: getAllByText(instance),
     getAllByProps: getAllByProps(instance),
+
+    queryByTestId: queryByTestId(instance),
```

Comment:
I think we should replace exports from `getBy API` and `queryBy API` by some factory function to avoid listing all the re-exports here. Eg.
```jsx
// getBy.js
...
export const getByAPI (instance) => ({
  queryByTestId: queryByTestId(instance),
  ...
});

// render.js
return {
  ...getByAPI(instance),
  ...queryByAPI(instance),
  ...
}
```